sdk-lambda/src/main/java/dev/restate/sdk/lambda/BaseRestateLambdaHandler.java

```diff
@@ -17,8 +17,11 @@
  * Base implementation of a Lambda handler to execute restate services
  *
  * <p>Implementation of <a href="https://aws.amazon.com/lambda/">AWS Lambda</a> {@link
- * RequestHandler} for serving Restate functions through <a
- * href="https://aws.amazon.com/api-gateway/">AWS API Gateway</a>.
+ * RequestHandler} for serving Restate functions.
+ *
+ * <p>Restate will invoke the Lambda using the same envelope of an API Gateway request/response. See
```

Comment:
```suggestion
 * <p>Restate can invoke Lambda functions directly or through API gateway. For both cases, it will invoke the Lambda using the same envelope of an API Gateway request/response. See
```